### PR TITLE
Loss Ops: Add Sigmoid Cross Entropy Loss

### PR DESCRIPTION
**Pull request by ManrajGrover**
#### Description
This PR adds `sigmoidCrossEntropy` loss to the API.

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1120)

### CODE REVIEW

**Comment by ManrajGrover** [comment]
Looks like test cases are [failing](https://travis-ci.org/tensorflow/tfjs-core/jobs/402702596#L681) for operations unrelated to this PR.

**Comment by ManrajGrover** [comment]
@dsmilkov Made the changes
